adr/0009-async-and-scheduled-jobs.md

```diff
@@ -0,0 +1,75 @@
+# 9. Async and scheduled jobs
+
+Date: 2019-11-07
+
+## Status
+
+Accepted
+
+## Context
+
+This service will need to process applications automatically and periodically in order to enforce time-dependent business rules and modify application status/state. Therefore, we need a way to run domain logic code outside the context of web requests, reliably and in a proactive/scheduled way.
```

Comment:
just `state` instead of `status/state` please, it's confusing enough as it is 🙂

Reply:
yes, could be read as both Rails app or an applicant's application (both work!)